language/move-lang/src/hlir/translate.rs

```diff
@@ -1607,14 +1620,32 @@ fn check_trailing_unit(context: &mut Context, block: &mut Block) {
         return;
     }
     match (&block[len - 2], &block[len - 1]) {
-        (sp!(loc, S::IfElse { if_block, else_block, ..}), trailing!(uloc))
-        | (sp!(loc, S::IfElse { if_block, else_block, ..}), trailing_returned!(uloc))
-            if divergent_block(if_block) && divergent_block(else_block) =>
-        {
+        (
+            sp!(
+                loc,
+                S::IfElse {
+                    if_block,
+                    else_block,
+                    ..
+                }
+            ),
+            trailing!(uloc),
+        )
```

Comment:
Some of these fmt changes make me sad, but I suppose it is what it is

Reply:
Yeah dang that is ugly. You could also annotate it with `#[rustfmt::skip]`

Reply:
Ah, I might do that after this lands. Thanks!

Reply:
Honestly that one does seem like a regression, feel free to fix + `rustfmt::skip` those blocks after it lands